js/ProgressView.windows.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import type {WindowsNativeProps} from './RNCProgressViewNativeComponent';
+import * as React from 'react';
+
+const RNCProgressView = requireNativeComponent('RNCProgressViewWindows');
+
+const styles = StyleSheet.create({
+  progressWindows: {
+    height: 5,
+  },
+});
+
+export default function ProgressViewWindows(props: WindowsNativeProps) {
+  const nativeProps = {
+    progressViewStyle: props.progressViewStyle,
+    progress: props.progress,
+    progressTintColor: props.progressTintColor,
+    trackTintColor: props.trackTintColor,
+    isIndeterminate: props.isIndeterminate,
+    progressImage: props.progressImage,
```

Comment:
Same comment as about the 'isIndeterminate' prop - did you, perhaps, forget to commit your example JS file?

Reply:
I've cloned the repo, and cannot see these props used anywhere, or in this PR.

Reply:
I didn't realize I was suppose to have an example JS file but that makes a lot of sense lol, I can add that! Would that go in the example app folder?

Reply:
You might not need to add a new file. If there already is one (and it looks like there is), you can add new examples to it.
Also, how did you obtain the screenshot in the PR description, if not by running JS code utilizing eg. the 'isIndeterminate' property?

Reply:
@tatianakapos If you had a separate app importing this module while testing, it'd be great to add the example usage from there to the example code in this repo.

Reply:
I changed the example app to test everything and to get the screenshot but I didn't want to change their example app so I just converted it back to the original after. Would you mind sending which file has the examples in it, I can't find it.

Reply:
I don't understand :) If you've previously modified the example app in this repo and then reverted your changes, just change that file again. 
I think there is a file in the example folder - your Windows CPP example app should be interacting with the JS file that's already in this repo.